cypress/e2e/submit_application_spec.cy.ts

```diff
@@ -0,0 +1,13 @@
+describe("submit application spec", () => {
+  it("successfully apply to one group", () => {
+    cy.login("webkom");
+    cy.visit("/");
+    cy.contains("Gå til søknad").click();
+    cy.contains("Velg komité").click();
+    cy.contains("Gå videre").click();
+    cy.get("input[name='phoneNumber']").type("12345678");
+    cy.contains("Søknadstekst").type("Hei jeg vil gjerne søke");
+    // cy.contains("Send inn søknad").click();
+    // cy.contains("Vi har mottatt søknaden din!").should("exist");
```

Comment:
Why is this commented?
Edit: nevermind, it's the CSRF thing right?

Reply:
Yup, that's the step that's being blocked by CSRF